pkgs/by-name/ni/nix-direnv/package.nix

```diff
@@ -1,34 +1,36 @@
-{ resholve, lib, coreutils, direnv, nix, fetchFromGitHub }:
+{
+  resholve,
+  lib,
+  coreutils,
+  nix,
+  fetchFromGitHub,
+  writeText,
+}:
 
 # resholve does not yet support `finalAttrs` call pattern hence `rec`
 # https://github.com/abathur/resholve/issues/107
 resholve.mkDerivation rec {
   pname = "nix-direnv";
-  version = "3.0.5";
+  version = "3.0.6";
 
   src = fetchFromGitHub {
     owner = "nix-community";
     repo = "nix-direnv";
     rev = version;
-    hash = "sha256-imUlc5v/Ow7dgjCjTyEeet+4lNHLeEwfqGQcB4dKcao=";
+    hash = "sha256-oNqhPqgQT92yxbKmcgX4F3e2yTUPyXYG7b2xQm3TvQw=";
   };
 
-  # skip min version checks which are redundant when built with nix
-  postPatch = ''
-    sed -i 1iNIX_DIRENV_SKIP_VERSION_CHECK=1 direnvrc
-  '';
-
   installPhase = ''
     runHook preInstall
-    install -m400 -D direnvrc $out/share/nix-direnv/direnvrc
+    install -m400 -D direnvrc $out/share/${pname}/direnvrc
     runHook postInstall
   '';
 
   solutions = {
     default = {
-      scripts = [ "share/nix-direnv/direnvrc" ];
+      scripts = [ "share/${pname}/direnvrc" ];
```

Comment:
Usually we advise contributors to avoid reusing pname, it creates a binding that goes too far. Prefer a string literal.